wordpress-mobile/WordPress-iOS
# History / Revisions: WordPressKit 1.4.3-beta.1 and new datamodel

This PR introduces:
- `WordPressKit 1.4.3-beta.1` with the new *revisions* property mapped for `RemotePost`.
- New *CoreData* model version *82* with the new *revisions* property for `AbstractPost` entity.
- `AbstractPost` model has a new *revisions* property.

**To test:**
- Run the app over any branch with CoreData model version 81
- CoreData migration should work with no problem

*Note:*
I need to add a new entity here before start the review
